deployment/oss-vdb-test/cloudbuild/gke/exporter/exporter.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: batch/v1
```

Comment:
Actually (can be for a future PR), can we move these GKE configs into `oss-vdb-test/k8s` instead living in this `cloudbuild` directory which is inaccurate?

Reply:
I'll just do this in this PR quickly